### ISSUE TITLE
eb -S output truncation yields wrong output

### ISSUE THREAD

**Issue by bryank-cs** [easybuilders/easybuild-framework#2394]
When searching for IGV with "eb -S IGV", the output improperly truncates the IGVTools directory name:

```
CFGS1=.../easyconfigs/i/IGV
 * $CFGS1/IGV-2.3.68-Java-1.7.0_80.eb
 * $CFGS1/IGV-2.3.80-Java-1.7.0_80.eb
 * $CFGS1/ools/IGVTools-2.3.68-Java-1.7.0_80.eb
 * $CFGS1/ools/IGVTools-2.3.72-Java-1.7.0_80.eb
 * $CFGS1/ools/IGVTools-2.3.75-Java-1.7.0_80.eb
```

**Comment by fgeorgatos** [comment]
good catch; in fact, this has never worked exactly in an excellent way, personally i'd rather see the following to happen:
* an EasyBuild modulefile defines $EBROOTEASYBUILD and could also provide $EB_EASYCONFIGS
* the output of `-S` should rather be defined against that $EB_EASOCONFIGS, where applicable
(choose your names, this is just an idea)

**Comment by scroogie** [comment]
Confirmed in EasyBuild 3.6.0. Seen when searching for Amber as well (cutting the Prefix).

**Comment by bryank-cs** [comment]
Not clear when, but this is fixed as of at least 4.4.0.
